src/core/Aggregators.java

```diff
@@ -34,7 +33,7 @@
     LERP,   /* Regular linear interpolation */
     ZIM,    /* Returns 0 when a data point is missing */
     MAX,    /* Returns the <type>.MaxValue when a data point is missing */
-    MIN     /* Returns the <type>.MinValue when a data point is missing */
+    MIN,    /* Returns the <type>.MinValue when a data point is missing */
```

Comment:
Hi @EaconTang yeah if you could revert that then squash all your commits into one please, that'd be great.